tests/integration-tests/tests/create/test_create/test_create_disable_sudo_access_for_default_user/pcluster.config.yaml

```diff
@@ -0,0 +1,41 @@
+Image:
+  Os: {{ os }}
+{% if scheduler == "slurm" %}
+LoginNodes:
+  Pools:
+    - Name: login
+      InstanceType: t2.micro
+      Count: 1
+      Networking:
+        SubnetIds:
+          - {{ public_subnet_id }}
+      Ssh:
+        KeyName: {{ key_name }}
+{% endif %}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Imds:
+    Secured: {{ imds_secured }}
+Scheduling:
+  Scheduler: {{ scheduler }}
+  {% if scheduler == "awsbatch" %}AwsBatchQueues:{% else %}SlurmQueues:{% endif %}
```

Comment:
I believe only "SlurmQueues"  is needed, right?

Reply:
Thanks this also makes me realise I need to remove it from other YAMLs